doc/lua_api.md

```diff
@@ -3877,6 +3885,12 @@ vectors are written like this: `(x, y, z)`:
 	* `min` and `max` are inclusive.
 	* If `min` is bigger than `max` on some axis, function always returns false.
 	* You can use `vector.sort` if you have two vectors and don't know which are the minimum and the maximum.
+* `vector.random_in_area(min, max)`:
+	* Returns a random integer position in area formed by `min` and `max`
+	* `min` and `max` are inclusive.
+	* You can use `vector.sort` if you have two vectors and don't know which are the minimum and the maximum.
+    * Don't use this function for making random direction vectors as it is
+      biased towards the unit cube corners, use `vector.random_direction` instead.
```

Comment:
Since this returns integers, why would anyone try to use it for generating random directions anyway?

Reply:
I could imagine modders doing something like `vector.random_in_area(vector.new(-1000, -1000, -1000), vector.new(1000, 1000, 1000)):normalize()` :smile:

(and indeed the unnecessarily increased discretization is another issue with this - though fun fact, lua random usually doesn't have a full mantissa of randomness either)

I agree though that it's probably just as fine to remove this note.